src/gtk/textctrl.cpp

```diff
@@ -797,8 +820,11 @@ bool wxTextCtrl::Create( wxWindow *parent,
         // new, empty control, see https://github.com/wxWidgets/wxWidgets/issues/11409
         gtk_entry_get_text((GtkEntry*)m_text);
 
+#ifndef __WXGTK3__
```

Comment:
Why is this added? Limiting border related stuff seems unrelated to max length.

Reply:
@MaartenBent ,
Will check.